lib/rules/no-restricted-imports.js

```diff
@@ -121,31 +139,128 @@ module.exports = {
         }
 
         /**
+         * Report a restricted path specifically for patterns.
+         * @param {node} node - representing the restricted path reference
+         * @returns {void}
+         * @private
+         */
+        function reportPathForPatterns(node) {
+            const importName = node.source.value.trim();
+
+            context.report({
+                node,
+                message: "'{{importName}}' import is restricted from being used by a pattern.",
+                data: {
+                    importName
+                }
+            });
+        }
+
+        /**
+         * Report a restricted path specifically for patterns.
```

Comment:
This is the same as the comment on the previous function. Is it supposed to be something different?

Reply:
Yep, bad copy/paste when refactoring that method out.  Thanks for the catch.  I must have read it 10 times and saw what I expected instead of what it actually is.  I will also add params and return this comment.

Reply:
Updated in the 8th commit.